opengrok-web/src/test/java/org/opengrok/web/api/v1/controller/SystemControllerTest.java

```diff
@@ -130,4 +136,26 @@ public void testDtagsEftarReload() throws IOException {
         // Cleanup
         IOUtils.removeRecursive(dataRoot);
     }
+
+    @Test
+    public void testIndexTime() throws IOException, ParseException {
+        Path dataRoot = Files.createTempDirectory("indexTimetest");
+        env.setDataRoot(dataRoot.toString());
+        File indexTimeFile = Paths.get(dataRoot.toString(), "timestamp").toFile();
+        indexTimeFile.createNewFile();
+        assertTrue(indexTimeFile.exists());
+        SimpleDateFormat f = new SimpleDateFormat("yyyy-MM-dd_hh:mm:ss+ZZ");
+        Date date = f.parse("2021-02-16_11:18:01+UTC");
+        indexTimeFile.setLastModified(date.getTime());
+
+        Response r = target("system")
+                .path("indextime")
+                .request().get();
+        String result = r.readEntity(String.class);
+
+        assertThat(result, containsString("2021-02-16T11:18:01.000+00:00"));
```

Comment:
Should not we check for equality?

Reply:
That's because JSON returns the string with double quotes and I thought checking the whole input is too strong. Thinking about this again, it should not hurt.